### PR TITLE
[CI:DOCS] Fix up example description of podman-image commands

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-image-mount.1.md [hunk 1]
```diff
@@ -32,20 +32,21 @@ Print the mounted images in specified format (json).
 
 ## EXAMPLE
 
+Mount multiple images. In rootless mode, execute podman unshare first:
```
Comment (vrothberg) [inline docs/source/markdown/podman-image-mount.1.md:35]: The second part on `podman mount` may confuse users.  I find it unclear whether the user needs to do that or if podman does it automatically.  Maybe extend an example for an obvious rootless example?  Here it's clearly a root directory.

File: docs/source/markdown/podman-image-mount.1.md [hunk 2]
```diff
@@ -32,20 +32,21 @@ Print the mounted images in specified format (json).
...
+List mounted images:
 ```
-podman mount
-
+podman image mount
 registry.fedoraproject.org/fedora:latest /var/lib/containers/storage/overlay/f3ac502d97b5681989dff84dfedc8354239bcecbdc2692f9a639f4e080a02364/merged
 registry.access.redhat.com/ubi8-init:latest /var/lib/containers/storage/overlay/0ff7d7ca68bed1ace424f9df154d2dd7b5a125c19d887f17653cbcd5b6e30ba1/merged
 ```
 
+List mounted images in json format:
```
Comment (vrothberg) [inline docs/source/markdown/podman-image-mount.1.md:49]:
```suggestion
List mounted images in JSON format:
```

File: docs/source/markdown/podman-image-mount.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/source/markdown/podman-image-mount.1.md:35]: Yes I hate the wording.   

How about?

Note: In rootless mode, image mounting works only after executing the `podman unshare` command to enter the user namespace.

**Comment by baude** [comment]
should we group these by subcommand or verb ? are you dead sure this is not going to end in a rebase hell ?

**Comment by rhatdan** [comment]
I grouped them together but it was too big. Now they are grouped by subcommand, but they should not cause conflicts unless someone else is touching example sections of man pages.  They should merge pretty quickly.

File: docs/source/markdown/podman-image-mount.1.md (hunk 1 above)
Comment (vrothberg) [reply docs/source/markdown/podman-image-mount.1.md]: LGTM

**Comment by vrothberg** [comment]
LGTM

**Review by giuseppe — APPROVED** [review]
/lgtm

**Pull request merged**